README.md

```diff
@@ -99,12 +104,13 @@ will produce firmware file `build/u2f.bin`.
 
 Supported targets are:
 
-- TOMU
-- MAPLE_MINI (STM32F103 board with push button on PB8)
-- BLUE_PILL (generic STM32F103 board without push button)
-- ST_DONGLE (ST-Link V2 clone, push button pulls down PA5/SWCLK pin)
+- TOMU http://tomu.im/
```

Comment:
http**s**://tomu.im/. Maybe it would look better if we turn each target itself into a link?
Something like this - [BALCK_PILL](https://wiki.stm32duino.com/index.php?title=Black_Pill)
```
[BALCK_PILL](https://wiki.stm32duino.com/index.php?title=Black_Pill)
```